build/requirements.txt

```diff
@@ -0,0 +1,8 @@
+pyproj==1.9.5.1
```

Comment:
You may want to not lock exact versions, but just major and minor, so the patched version (the third number) is flexible. If they are using SemVar then those are only bug fixes (https://semver.org/).